app/main/containers/Search/index.js

```diff
@@ -109,7 +110,9 @@ class Search extends Component {
     // NOTE: when page refreshed (location.reload) componentWillUnmount is not called
     window.addEventListener('beforeunload', this.cleanup)
     this.electronWindow.on('show', this.focusMainInput)
+    this.electronWindow.on('show', this.updateElectronWindow)
     this.electronWindow.on('show', trackShowWindow)
+    this.electronWindow.on('move', this.moveElectronWindow)
```

Comment:
Maybe instead of `on('move') + debounce` you can use [moved event]( https://github.com/electron/electron/blob/master/docs/api/browser-window.md#event-moved-macos)